internal/provider/nutanix/ignition.go

```diff
@@ -35,6 +35,15 @@ func (p nutanixProvider) PostCreateManifestsHook(_ *common.Cluster, _ *[]string,
 		return fmt.Errorf("error deleting machineset: %w", err)
 	}
 
+	// Delete control-plane-machine-set
```

Comment:
Do we need to do it only in OCP >= 4.14?

Reply:
Basically yes but `control-plane-machine-set` is present only since 4.14

Reply:
what will happen in 4.12? we will receive an error?

Reply:
No, just an empty file list